library-best-practices/JLBP-5.md

```diff
@@ -1,20 +1,30 @@
 [JLBP-5] Avoid dependencies that overlap classes with other dependencies
 ------------------------------------------------------------------------
 
-- Definition: When two artifacts (with either a distinct group or distinct
+Definition: When two artifacts (with either a distinct group or distinct
   artifact id) "overlap classes," that means that there is at least one
   fully qualified class name provided by both of the artifacts.
-- This most commonly happens when a library has published the same classes in
-  artifacts with different group ID or artifact IDs. Always use the primary variant.
-- Example 1: Guava's main artifact is `guava`, but from versions 13.0 to 17.0,
+
+This commonly happens when a library has published the same classes in
```

Comment:
Since we have three scenarios where this happens, I think it would help readability to convert to a numbered or bulleted list for the scenarios.